packages/spear-cli/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@spearly/spear-cli",
-  "version": "1.1.12",
+  "version": "1.1.14",
```

Comment:
Maybe even `1.2.0` since it's a big feature addition. 😄 
(You don't have to do it.)

Reply:
That's true.
I'll change this version to `v1.2.0` when releasing it 🚀